backend/src/main/java/eu/bbmri_eric/negotiator/notification/NotificationController.java

```diff
@@ -28,4 +26,38 @@ public List<NotificationDTO> getNotificationsForUser() {
     return userNotificationService.getNotificationsForUser(
         AuthenticatedUserContext.getCurrentlyAuthenticatedUserInternalId());
   }
```

Comment:
Not sure the notification Controller is the right place for this. Shouldn't it be separate?

Reply:
Yes your are correct, the functionality was seperated from the contoller